src/main/java/edu/harvard/iq/dataverse/authorization/providers/builtin/DataverseUserPage.java

```diff
@@ -284,6 +284,12 @@ public void validateNewPassword(FacesContext context, UIComponent toValidate, Ob
 
     public String save() {
         boolean passwordChanged = false;
+        
+        //First reget user to make sure they weren't deactivated or deleted
+        if (session.getUser().isAuthenticated() && !session.getUser(true).isAuthenticated()) {
+            return "dataverse.xhtml?alias=" + dataverseService.findRootDataverse().getAlias() + "&faces-redirect=true";
```

Comment:
why not simplify and just go to homepage, i.e. "/"

Reply:
I can make that change. i was just using the pattern from the bottom of the save method.

Reply:
I'm going to leave it as is because when I tried return "/"; it left me in edit mode on the user page which I think would be confusing to the deleted/deactivated user.